lib/cluster.rb

```diff
@@ -134,11 +136,11 @@ def ocns=(ocns)
   end
 
   def ocn_resolutions
-    Clusterable::OCNResolution.with_ocns(ocns)
+    @ocn_resolutions ||= Clusterable::OCNResolution.with_ocns(ocns).to_a
   end
 
   def ht_items
-    Clusterable::HtItem.with_ocns(ocns)
+    @ht_items ||= Clusterable::HtItem.with_ocns(ocns).to_a
```

Comment:
These memoizations aren't necessarily required for the cost report as it stands now given that we are building htitems from the solr records and applying directly to the cluster, but I don't think they're a problem.